app/javascript/react/containers/LandingPage.js

```diff
@@ -1,48 +1,74 @@
-import React from 'react'
+import React, { useState } from 'react'
 import ContactForm from "../components/landing_page/ContactForm"
 import StepTile from "../components/landing_page/StepTile"
 import WeekCalendar from 'react-week-calendar';
 
 
 const LandingPage = (props) => {
+  const[hiddenFields, setHiddenFields] = useState(true)
 
-const stepData=[
-  {
-    id:1,
-    image:"https://i.imgur.com/MRuIRf0.png",
-    description:"blah"
-  },
-  {
-    id:2,
-    image:"https://i.imgur.com/MRuIRf0.png",
-    description:"blah"
-  },
-  {
-    id:3,
-    image:"https://i.imgur.com/MRuIRf0.png",
-    description:"blah"
-  }
+  const stepData=[
+    {
+      id:1,
+      image:"https://i.imgur.com/MRuIRf0.png",
+      description:"blah",
+      class:"pics2"
+    },
+    {
+      id:2,
+      image:"https://i.imgur.com/MRuIRf0.png",
+      description:"blah",
+      class:"pics2"
+    },
+    {
+      id:3,
+      image:"https://i.imgur.com/MRuIRf0.png",
+      description:"blah",
+      class:"pics2"
+    }
+  ]
```

Comment:
you should still avoid using hardcoded dummy data in your PRs and commits! only deliverable data that you would be satisfied being live